tests/features/step_definitions/10210.ACs.feature.steps.intTest.ts

```diff
@@ -77,7 +77,7 @@ defineFeature(feature, (test) => {
   //   });
   //   then("I am given the 400 error code", () => {
   //     expect(response.status).toEqual(400);
-  //     expect(response.body.errors).toContain('"fuelPropulsionSystem" must be one of [DieselPetrol, Hybrid, Electric, CNG, Fuel cell, LNG, Other]');
+  //     expect(response.body.errors).toContain('"fuelPropulsionSystem" must be one of [DieselPetrol, Diesel, Petrol, Hybrid, Electric, CNG, Fuel cell, LNG, Other]');
```

Comment:
Is there any point to this if the tests are commented out?

Reply:
To be fair probably not, I was mostly just being pedantic if we're honest